python/ray/serve/tests/test_logging.py

```diff
@@ -280,6 +281,29 @@ def check_log_file(log_file: str, expected_regex: list):
         check_log_file(resp2["log_file"], user_class_method_log_regexes)
 
 
+def test_extra_field(serve_and_ray_shutdown):
+    """Test ray serve extra logging"""
+    ray.init(runtime_env={"env_vars": {"RAY_SERVE_ENABLE_JSON_LOGGING": "1"}})
```

Comment:
What happens if `RAY_SERVE_ENABLE_JSON_LOGGING` is not enabled and someone uses the extra fields? they're ignored?

Reply:
nothing happens, we just ignore.

this aligns with the current logging lib behavior. 
```
The fourth keyword argument is extra which can be used to pass a dictionary which is used to populate the __dict__ of the [LogRecord](https://docs.python.org/3/library/logging.html#logging.LogRecord) created for the logging event with user-defined attributes
```

https://docs.python.org/3/library/logging.html#logging.debug